src/domain/users/errors.ts

```diff
@@ -0,0 +1,14 @@
+import { DomainError, ErrorLevel } from "@domain/shared"
```

Comment:
This file needs to be added to `src/app/errors.ts` so that the linter catches missing error types in `src/graphql/error-map.ts`